index.html

```diff
@@ -61,14 +61,138 @@ <h4 class="heading">Skills</h4>
           </div>
         <hr class="horizontal"/>
           <div class="icon">
-            <ul>
+            <ul class="icon-list">
               <li><a href='#' ><i class='fab fa-twitter' ></i></a></li>
               <li><a href='#' ><i class='fab fa-linkedin-in' ></i></a></li>
               <li><a href='#' ><i class='fab fa-medium' ></i></a></li>
               <li><a href='#' ><i class='fab fa-github' ></i></a></li>
               <li><a href='#' ><i class='fab fa-angellist' ></i></a></li>
             </ul>
             </div>
+            <div class="nav-arrow">
+              <a href="#work"
+                ><img class="arrow" src="images/arrow.png" alt="" />
+              </a>
+              <a href="#work" class="works"><p>Work</p></a>
+            </div>
+      </section>
+      <section class="grid-container">
```

Comment:
- [ ] Kindly check on your design, there is an empty space left on the right, also the background images are not well aligned kindly check the comparison below with the Figma design:


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
<img width="172" alt="image" src="https://user-images.githubusercontent.com/61408860/177737970-19804bec-f876-45e4-a546-da7a34396c2e.png"> |  ![image](https://user-images.githubusercontent.com/61408860/177738084-ab779fb3-0c4f-42e7-b44d-f8bb75592e1f.png)

- [ ] Kindly ensure your buttons are clickable and follow the design below:

![image](https://user-images.githubusercontent.com/61408860/177738376-c2b71dc5-b079-4228-98ee-dbc884b92c32.png)